cli/deployment/order.go

```diff
@@ -0,0 +1,137 @@
+package deployment
+
+import (
+	"errors"
+	"fmt"
+	"log"
+)
+// Order function receives map of configs with Config.FullName() string as a key,
+// find dependencies between them, and order them topologically using directed graph.
+// Returns array of ordered config's objects and error if configs have circle dependencies
```

Comment:
"The order function receives a map of Configs keyed by each configuration's FullName, resolves dependencies between them, and sorts them topologically. It returns the sorted array of Configs, or an error if dependencies are cyclic."